### DISCUSSION TITLE
Welcome to Website-Template Discussions!

### DISCUSSION THREAD

**Discussion by Diptenusarkar** [Diptenusarkar/Website-Template#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aryav-022** [comment]
What is the issue i can work on... and I am new to hacktoberfest, is this a valid repo to contribute in hactoberfest?

**Reply by Diptenusarkar** [reply]
Hello Aryav, Glad you are new to hacktoberfest.
Unfortunately, this rep doesn't have any problem, 
This is a website template, you can do one thing just create any type of template then push your pull request. That's all.
Best of luck, See you soon with your code 🤞

**Comment by DYNAMICMORTAL** [comment]
_Thank You @Diptenusarkar. It was wonderful, working with you. I will continue to commit to this project in the near future. Thank you until then._

**Reply by Diptenusarkar** [reply]
It was my pleasure, To get a humble contribution. Thank You so much, See you soon. 
Happy code 🤞
